arch/arm64/boot/dts/qcom/sdm636-xiaomi-tulip.dts

```diff
@@ -631,24 +503,13 @@
 };
 
 &usb3_dwc3 {
-	status = "okay";
-
 	dr_mode = "peripheral";
-};
 
-&venus {
 	status = "okay";
+};
 
+&venus {
 	firmware-name = "qcom/venus-4.4/venus.mdt";
-};
 
-&wifi {
```

Comment:
Why did you remove wi-fi node? Even if it is not fully working, we can still have it here, I think.

I also think you're using kernel config which is different from mine, can I see your defconfig? (Run `make savedefconfig`and share defconfig file somewhere? To be precisefull command will look something like `make ARCH=arm64 CROSS_COMPILE=aarch64-alpine-linux-musl- savedefconfig`)